src/components/NewsGroup.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DateNewsCard, IconHeading, Button } from '@baltimorecounty/dotgov-components';
+const NewsGroup = (props) => {
+	const { Newsitems = [] } = props;
+	return (
+		<React.Fragment>
+            <div className="row">
+				{Newsitems.map((Newsitem) => (
+                    <div className="col">
+					<DateNewsCard
+						key={Newsitem.id}
+						date={Newsitem.date}
+						headline={Newsitem.headline}
+						snippet={Newsitem.snippet}
+						link={Newsitem.link}
+					/>
+                    </div>
+				))}
+			</div>
+			 <div  style={{ display: 'flex', justifyContent: 'center', padding: '40px' }}>
```

Comment:
in order to use this in site executive we want to avoid specifying inline styles. Let me create helper class in the components project and get that back to you.